LiveSplit.AutoSplitters.xml

```diff
@@ -12675,4 +12675,15 @@
         <Description>Auto Splitting available. (By DevilSquirrel)</Description>
         <Website>https://github.com/ShootMe/LiveSplit.CrankedUp</Website>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Omno</Game>
+        </Games>
+        <URLs>
+            <URL>https://github.com/Istador/ASL-Omno/blob/public/Omno.asl</URL>
```

Comment:
You need to use the raw file link instead.

Reply:
Fixed, thank you.